core/jvm/src/LocalDate.kt

```diff
@@ -39,6 +39,9 @@ public actual class LocalDate internal constructor(internal val value: jtLocalDa
         public actual fun fromEpochDays(epochDays: Int): LocalDate =
             LocalDate(jtLocalDate.ofEpochDay(epochDays.toLong()))
```

Comment:
Minor: I'd write this as `fromEpochDays(epochDays.toLong())` to avoid the tiny bit of unnecessary code duplication. Maybe this goes for `toEpochDays()`, though there, the benefit is even more miniscule. Up to you.